llama_index/evaluation/base.py

```diff
@@ -192,10 +194,10 @@ def evaluate_source_nodes(self, response: Response) -> List[str]:
 
             if "yes" in raw_response_txt.lower():
                 response_txt = "YES"
-            elif "no" in raw_response_txt.lower():
-                response_txt = "NO"
             else:
-                raise ValueError("The response is invalid")
+                response_txt = "NO"
```

Comment:
not for this PR: we should consider using the output parser abstraction to write these logic.